controllers/chat/sendSignedMessage.js

```diff
@@ -131,8 +132,44 @@ const sendSignedMesage = async (req, res) => {
       });
     }
 
-    const chat = await channel.sendMessage(baseMessage);
+    const chat = await channel.sendMessage(baseMessage, {skip_push: true});
+    let notificationPayload = {
+      title: `${senderInfo?.user?.username}`,
+      body: `${message.substring(0, 100)}`
+    };
+    let dataPayload = {
+      channel_id: channelId,
+      user_id: req.userId,
+      messages_id: chat.message.id,
+      message,
+      message_schema: currentMessageType,
+      type: 'message.new',
+      status: 'sent',
+      is_big_message: 'false',
+      attachment: JSON.stringify(attachments) ?? '',
+      created_at: chat.message.created_at,
+      is_annoymous: 'false'
+    };
+    const dataSizeInBytes = Buffer.byteLength(JSON.stringify(dataPayload), 'utf8');
+    const dataSizeInKilobytes = dataSizeInBytes / 1024;
+    if (dataSizeInKilobytes > 4) {
+      dataPayload.is_big_message = 'true';
+      dataPayload.attachment = '';
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The notification payload and data payload are created here. However, there's no validation of the input data before using them. This could lead to potential issues if the input data is not as expected. Consider adding validation for the input data before using them.
Also, the `is_big_message` field is a string instead of a boolean. Consider changing it to a boolean for consistency and to avoid potential type-related issues.
```javascript
let notificationPayload = {
  title: `${senderInfo?.user?.username}`,
  body: `${message.substring(0, 100)}`
};

if (!notificationPayload.title || !notificationPayload.body) {
  // Handle case when notification payload is not valid
}

let dataPayload = {
  channel_id: channelId,
  user_id: req.userId,
  messages_id: chat.message.id,
  message,
  message_schema: currentMessageType,
  type: 'message.new',
  status: 'sent',
  is_big_message: false,
  attachment: JSON.stringify(attachments) ?? '',
  created_at: chat.message.created_at,
  is_annoymous: 'false'
};

const dataSizeInBytes = Buffer.byteLength(JSON.stringify(dataPayload), 'utf8');
const dataSizeInKilobytes = dataSizeInBytes / 1024;
if (dataSizeInKilobytes > 4) {
  dataPayload.is_big_message = true;
  dataPayload.attachment = '';
}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->